package.json

```diff
@@ -270,7 +270,7 @@
     "eslint-plugin-import": "^2.23.0",
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "eslint-plugin-prettier": "^3.4.0",
-    "eslint-plugin-react": "^7.23.0",
+    "eslint-plugin-react": "^7.24.0",
```

Comment:
I had to update it. Otherwise, it failed to apply the patch on cached dependencies on the CI